### ISSUE TITLE
Compile fails with 'class Shader' has no member named 'has_parameter'

### ISSUE THREAD

**Issue by Asgatoril** [Zylann/godot_voxel#482]
The method ` bool Shader::has_parameter(const StringName &p_name) ` was removed and replaced with `Array Shader::_get_shader_uniform_list(bool p_get_groups)` with this commit:

https://github.com/godotengine/godot/commit/7dbc458bb4f3e0cc94e5070bd33bde41d214c98d

This causes the compilation process of godot_voxel to run into this error:

```
modules/voxel/edition/voxel_tool_lod_terrain.cpp: In function 'Array zylann::voxel::separate_floating_chunks(VoxelTool&, zylann::Box3i, Node*, Transform3D, Ref<VoxelMesher>, Array)':
modules/voxel/edition/voxel_tool_lod_terrain.cpp:595:67: error: 'class Shader' has no member named 'has_parameter'
  595 |                                                 sm->get_shader()->has_parameter(VoxelStringNames::get_singleton().u_block_local_transform)) {
      |                                                                   ^~~~~~~~~~~~~


```

**Comment by MGilleronFJ** [comment]
I have some code to fix this locally. It wasnt as simple to fix because using `get_shader_uniform_list` is way more expensive than the StringName cache lookup there was before.

**Comment by Zylann** [comment]
Should be fixed in 98e4c92561836ab0994ca3a38443d05ccc76c72e

**Comment by Asgatoril** [comment]
Works again.
Thank you :)
